apps/app-article/src/index.html

```diff
@@ -8,6 +8,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" type="text/css" href="app-article.less" />
```

Comment:
```suggestion
    <link rel="stylesheet" type="text/css" href="../../less/app-article.less" />
```

Reply:
changed according to your last comment :)